pkg/controller/useraccount/useraccount_controller_test.go

```diff
@@ -1133,6 +1220,164 @@ func TestDisabledUserAccount(t *testing.T) {
 	})
 }
 
+func TestLookupAndDeleteCheUser(t *testing.T) {
```

Comment:
Isn't there some overlap between the `TestLookupAndDeleteCheUser` tests here and the `TestReconcile` tests above?

Reply:
I think only a little bit, so figured it's not harmful. 🙂

Reply:
I would have preferred all tests in the same `Test..` func, but that's because I thought the overlap was bigger. My point is, if we need to add tests, we'll have to figure out what's the best location for them.